frosty_flutter/example/pubspec.yaml

```diff
@@ -5,12 +5,13 @@ publish_to: 'none'
 version: 1.0.0
 
 environment:
-  sdk: ^3.2.0
+  sdk: ^3.3.0
 
 dependencies:
   flutter:
     sdk: flutter
-  frosty_flutter: ^3.0.0
+  frosty_flutter:
+    path: ..
```

Comment:
Required for local development but shouldn't be included when merging for release.